src/items.rs

```diff
@@ -1905,12 +1905,32 @@ fn get_missing_arg_comments(
 
 impl Rewrite for ast::Param {
     fn rewrite(&self, context: &RewriteContext<'_>, shape: Shape) -> Option<String> {
+        let param_attrs_result = self
+            .attrs
+            .to_vec()
```

Comment:
Nit: if I understand correctly, we do not need to call `to_vec()` here.

Reply:
Yup, you are correct!